division3_1.py

```diff
@@ -0,0 +1,7 @@
+"""Write a Python program to find those numbers which are divisible by 7 and multiple of 5,
+between 1500 and 2700 (both included).
+"""
+
+for i in range(1500, 2701, 1):
+    if i % 7 == 0 and i % 5 == 0:
+        print(i)
```

Comment:
No need to add counter param for range. Range will go from first value to to last one.